assets/js/src/core/modules/element/editor/shared-tab-manager/tabs/properties/properties-container.styles.tsx

```diff
@@ -13,27 +13,14 @@
 
```

Comment:
Same here, can we use our utility classes in that case? 
Because then we can delete this file completely